Java/TernarySearch.java

```diff
@@ -0,0 +1,42 @@
+// Ternary search 
```

Comment:
- Update java folder readme with your entry

Reply:
Updated the readme file. Kindly check, if any other changes required then let me know.